curator-test/src/site/confluence/index.confluence

```diff
@@ -1,10 +0,0 @@
-h1. Testing Utilities
```

Comment:
This is merged  with "utilities.confluence" to https://curator.apache.org/docs/utilities. That is good! It is a subset of that page.